model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -147,17 +143,34 @@ def run(self, metrics):
                     # Failure
                     return 1
 
-                if process_killed:
-                    continue
-
                 if process.returncode != 0:
                     if self._output.find(
-                            "Please use a larger time window.") > 0:
-                        self._config['measurement-interval'] += INTERVAL_DELTA
-                        logger.info(
-                            "perf_analyzer's measurement window is too small, "
-                            f"increased to {self._config['measurement-interval']} ms."
-                        )
+                            "Failed to obtain stable measurement") > 0:
```

Comment:
This section is tested by `L0_perf_analyzer` , Is there anything could you add to the tests there?

Reply:
I'll add a unittest for it.